kasa/tests/common.py

```diff
@@ -0,0 +1,102 @@
+import glob
```

Comment:
`common` is a very non-descriptive name, perhaps as such its contents belong to the main conftest.py? And maybe `FixtureInfo` and functions related to it could be moved to `fixtureinfo.py`?

Reply:
Done